pypolychord/polychord.py

```diff
@@ -225,10 +226,430 @@ def wrap_prior(cube, theta):
+        Only available in Python interface.
+        shape (:, nDims)
+    paramnames: List [(string,string)]
+        (Default: None)
+        Mapping of label:Tex for all parameters in order. E.g. for two physical
+        parameters and one derived:
+        paramnames = [("p0", "$p_0$"), ("p1", "$p_1$), ("d0", "$d_0$)]
+
+    Returns
+    -------
+
+    anesthetic.NestedSamples(root=<base_dir>/<file_root>)
+
+    In general the contents of <base_dir> is a set of getdist compatible files.
+
+    <root> = <base_dir>/<file_root>
+
+    <root>.txt                                              (posteriors = True)
+        Weighted posteriors. Compatible with getdist. Each line contains:
+
+          weight, -2*loglikelihood, <physical parameters>, <derived parameters>
+
+        Note that here the weights are not normalised, but instead are chosen
+        so that the maximum weight is 1.0.
+
+    <root>_equal_weights.txt                                    (equals = True)
+        Weighted posteriors. Compatible with getdist. Each line contains:
+            1.0, -2*loglikelihood, <physical parameters>, <derived parameters>
+
+    <root>_dead.txt                                         (write_dead = True)
+        Dead points. Each line contains:
+            loglikelihood, <physical parameters>, <derived parameters>
+
+    <root>_phys_live.txt                                    (write_live = True)
+        Live points. Each line contains:
+            <physical parameters>, <derived parameters>, loglikelihood
+        Mostly useful for run-time monitoring.
+
+    <root>.resume
+        Resume file. Human readable.
+
+    <root>.stats
+        Final output evidence statistics
+
+    """
+
+    try:
+        from mpi4py import MPI
+        comm = MPI.COMM_WORLD
+        rank = comm.Get_rank()
+    except ImportError:
+        rank = 0
+
+    paramnames = kwargs.pop('paramnames', None)
+
+    default_kwargs = {
+        'nDerived': 0,
+        'prior': default_prior,
+        'dumper': default_dumper,
+        'nlive': nDims*25,
+        'num_repeats': nDims*5,
+        'nprior': -1,
+        'nfail': -1,
+        'do_clustering': True,
+        'feedback': 1,
+        'precision_criterion': 0.001,
+        'logzero': -1e30,
+        'max_ndead': -1,
+        'boost_posterior': 0.0,
+        'posteriors': True,
+        'equals': True,
+        'cluster_posteriors': True,
+        'write_resume': True,
+        'write_paramnames': False,
+        'read_resume': True,
+        'write_stats': True,
+        'write_live': True,
+        'write_dead': True,
+        'write_prior': True,
+        'maximise': False,
+        'compression_factor': np.exp(-1),
+        'synchronous': True,
+        'base_dir': 'chains',
+        'file_root': 'test',
+        'grade_dims': [nDims],
+        'nlives': {},
+        'seed': -1,
+    }
+    default_kwargs['grade_frac'] = [1.0]*len(default_kwargs['grade_dims'])
```

Comment:
I understood that `grade_frac` needed to have the same length as `grade_dims` (copied from the original interface)

Reply:
Just realised that this isn't quite right, since the default `grade_frac` should the same length as the input `grade_dims`, not the default `grade_dims`